docs/source/getting-started/model-registry/step3-model-version-alias.rst

```diff
@@ -0,0 +1,77 @@
+Alias Model Version
+===================
+
+Model version aliases provide a flexible way to set named aliases on model versions. For example, 
```

Comment:
Can we mention which version of MLflow this is supported in? As this is a new feature, we want to make sure that users that are looking at the 'latest' version of the docs don't think that this is supported in some LTS MLR version.